src/main/java/com/wavefront/integrations/FDBMetricsReporter.java

```diff
@@ -63,6 +65,8 @@
 
     private List<String> disabledMetrics = new ArrayList<>();
 
+    private long metricTimestamp = System.currentTimeMillis();
```

Comment:
You can remove this and pass null to the sender. The timestamp will be automatically added.

Reply:
done